docs/blog/introducing-weps.rst

```diff
@@ -0,0 +1,81 @@
+Introducing Write the Docs Enhancement Proposals
+================================================
+
+Write the Docs has been growing each year for the past 7,
+and the scope of the community has expanded dramatically.
+We started with a single conference,
+and now we have multiple conferences,
+over 50 meetups around the world,
+multiple online spaces,
+and many other ongoing projects.
+
+We have often followed open source in our approach to things,
+keeping a loose governance model and mostly letting the work speak for itself.
+However,
+we've been noticing a number of growing pains with this approach.
+It makes it much harder for new people to understand and join the community in leadership positions,
+since they don't understand how the community actually works.
+Even folks doing a lot of work in the community are often lost or confused when it comes to other ongoing projects.
```

Comment:
How about elaborating what we refer to when we refer to "ongoing projects"?

e.g., the job board, salary survey, newsletter, etc ...

Reply:
I've updated this copy to make it a bit clearer. It was more trying to explain that folks in one part of the organization don't necessarily know what other folks are working on. 👍